sct-commons/src/main/java/org/lfenergy/compas/sct/commons/scl/SclRootAdapter.java

```diff
@@ -242,8 +244,20 @@ public DataTypeTemplateAdapter getDataTypeTemplateAdapter(){
      * @throws ScdException throws when unknown IED
      */
     public IEDAdapter getIEDAdapterByName(String iedName) throws ScdException {
```

Comment:
Why 2 methods? getIEDAdapterByName and findIedAdapterByName are very close

Reply:
I need a method that does not throw an exception if IED does not exists. That's why I created `findIedAdapterByName` which return an Optional. I don't want the automated process to stop on the first error, because I need to collect all errors.

`getIEDAdapterByName` throws an exception and is widely used (more than 10 times) so It is useful and should not be deleted.

I think it is OK to have both. Like in JPA where you have `getOne` which throws an exception, and `findById` which never throws exception.